pandas/tests/reshape/merge/test_merge.py

```diff
@@ -2209,6 +2209,10 @@ def test_merge_series(on, left_on, right_on, left_index, right_index, nm):
         (0, 0, {"suffixes": ("_a", None)}, ["0_a", 0]),
         ("a", "a", {}, ["a_x", "a_y"]),
         (0, 0, {}, ["0_x", "0_y"]),
+        (0, 0, {"suffixes": None}, [0, 0]),
+        (0, 0, {"suffixes": (None, None)}, [0, 0]),
+        ("a", "a", {"suffixes": None}, ["a", "a"]),
+        ("a", "a", {"suffixes": (None, None)}, ["a", "a"]),
```

Comment:
can you add the tests for partial suffiex e.g. `(None, 'a')`` for example as i think these are removed elsewhere

Reply:
those tests exists already, likes 2205 and 2208